packages/transactions/src/TokenRedeemV1.ts

```diff
@@ -0,0 +1,108 @@
+import proto from '@helium/proto'
+import Transaction from './Transaction'
+import {
+  toUint8Array, EMPTY_SIGNATURE, toAddressable, toNumber,
+} from './utils'
+import { Addressable, SignableKeypair } from './types'
+
+interface Options {
+  account?: Addressable
+  amount?: number
+  fee?: number
+  nonce?: number
+  tokenType?: number
+  signature?: Uint8Array
+}
+
+interface SignOptions {
+  keypair: SignableKeypair
+}
+
+export default class TokenRedeemV1 extends Transaction {
+  public account?: Addressable
+
+  public amount?: number
+
+  public fee?: number
+
+  public nonce?: number
+
+  public tokenType?: number
+
+  public signature?: Uint8Array
+
+  public type: string = 'token_redeem_v1'
```

Comment:
Should this be public? In what other case would the type not be `'token_redeem_v1'` ?